source/extensions/filters/http/oauth2/filter.h

```diff
@@ -94,6 +94,27 @@ struct FilterStats {
   ALL_OAUTH_FILTER_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+/**
+ * Helper structure to hold custom cookie names.
+ */
+struct CookieNames {
+  CookieNames(
+      const envoy::extensions::filters::http::oauth2::v3alpha::OAuth2Credentials::CookieNames&
+          cookie_names)
+      : CookieNames(cookie_names.bearer_token(), cookie_names.oauth_hmac(),
+                    cookie_names.oauth_expires()) {}
+
+  CookieNames(const std::string& bearer_token, const std::string& oauth_hmac,
+              const std::string& oauth_expires)
+      : bearer_token_(bearer_token.empty() ? "BearerToken" : bearer_token),
+        oauth_hmac_(oauth_hmac.empty() ? "OauthHMAC" : oauth_hmac),
+        oauth_expires_(oauth_expires.empty() ? "OauthExpires" : oauth_expires) {}
+
+  std::string bearer_token_;
+  std::string oauth_hmac_;
+  std::string oauth_expires_;
```

Comment:
can they be const? :D

Reply:
Ah, thanks. I think I missed setting these to `const` waiting if I ever had to set each of these properties directly 🙂, but forgot to revisit. Thank you again.